notebooks/official/model_monitoring/batch_prediction_model_monitoring.ipynb

```diff
@@ -0,0 +1,800 @@
+{
```

Comment:
<p>cloud service -&gt; Cloud AI service</p><p> </p><p>Run the next cell to import this model into your project -&gt; Next, import the model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>"cloud service -&gt; Cloud AI service" - but BQ is not a Cloud AI service so leaving as is. </p><p> </p><p>Other comment fixed.</p><p> </p><p> </p>